appinventor/components/src/com/google/appinventor/components/annotations/UsesNativeLibraries.java

```diff
@@ -27,4 +27,7 @@
    */
   String libraries() default "";
   String v7aLibraries() default "";
+  
+  String[] libraries() default {};
```

Comment:
These two entries conflict with those specified above and prevent the system from compiling.

Reply:
Should I revert changes made to this file as I can't find a way to use the array version without breaking the old ones?

Reply:
Since we do not use the `@UsesNativeLibraries` annotation, I think it is okay to go without here. Alternatively, rename them to `libraryList()` and `v7aLibraryList()`.